L/Libtask/build_tarballs.jl

```diff
@@ -1,52 +1,58 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using Pkg
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
-name = "Libtask"
-version = v"0.3.2"
-commit_id = "fbe338053f402d76524d0a01f3796dd7da90b781"
+julia_version = v"1.3.1"
 
-# see https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/336
-# ENV["CI_COMMIT_TAG"] = ENV["TRAVIS_TAG"] = "v" * string(version)
+name = "Libtask"
+version = VersionNumber("0.4.0-julia.$(julia_version.major).$(julia_version.minor)")
 
+# Collection of sources required to build Libtask
 sources = [
-    "https://github.com/TuringLang/Libtask.jl.git" => commit_id,
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
-script_file = joinpath(@__DIR__, "build_dylib.sh")
-if !isfile(script_file) # when run generate_buildjl.jl
-    script_file = joinpath(@__DIR__, "L/Libtask/build_dylib.sh")
-end
-script = read(script_file, String)
+script = raw"""
+# create output directory
+mkdir -p "${libdir}"
+
+# compile library
+cd "${WORKSPACE}/srcdir/"
+export CFLAGS="-I${includedir} -I${includedir}/julia -O2 -shared -std=gnu99 -fPIC"
+if [[ "${target}" == *-mingw* ]]; then
+  export LDFLAGS="-ljulia -lopenlibm"
```

Comment:
```suggestion
  export LDFLAGS="-L${prefix}/bin -ljulia -lopenlibm"
```
might help as DLLs are kept in `bin` not `lib` (due to limitations of the Windows dynamic linker)

Reply:
I'd prefer to use the platform-independent `libdir`

Reply:
I added `libdir`.